packages/desktop-client/src/components/modals/CreateEncryptionKey.js

```diff
@@ -91,6 +91,11 @@ export default function CreateEncryptionKey({
                     will need to reset them. Actual will automatically take you
                     through this process.
                   </li>
+                  <li>
+                    If you reuse the password you log in to the server here,
+                    anybody else who has the server password will be able to
+                    decrypt your data.
+                  </li>
```

Comment:
```suggestion
                  <li>
                    It is recommended for the encryption password to be different than the log-in password in order to better protect your data.
                  </li>
```

Reply:
Ah right, theoretically a malicious server could log your password and use it to decrypt the file. So reuse would throw out all the security guarantees e2e offers 👍